client/src/index.css

```diff
@@ -626,6 +626,28 @@ header.searchPagination {
 	margin-bottom: 0;	
 	border-radius: 0;
 	border: 1px solid transparent;
+	animation: flash 1.25s forwards linear normal;
+}
```

Comment:
can we add
	animation-iteration-count: 5;

Reply:
Maybe this is less "in your face"
animation-iteration-count: 3;

Reply:
I'd suggest flashing 3 times (feels like it grabs enough attention), it could become annoying when it flashes 5 times.